### ISSUE TITLE
After reboot Homey thinks it is part of another Z-Wave network

### ISSUE THREAD

**Issue by Corvo** [athombv/homey#789]
**What is your Homey version (Settings → System)?**

0.9.2

**What did you try to achieve?**

Since Homey was no longer noticing the usage of my KaKu remote, I decided to restart Homey in order to get KaKu fully working again.
(FYI: Homey was still able to send KaKu signals, but simply didn't "hear" any signals anymore.)

**What was the result?**

Everyhing KaKu related was working (sending and "hearing" is okay again).
However after the reboot Homey states it is now _part of an other Z-Wave network_.
This prevents me from adding new Z-Wave devices and my Fibaro Multisensor no longer reports anything (except lux). Removing Homey from this other z-wave network is impossible, since this other controller does not exist...

**What did you expect as result?**

That after the reboot, Homey would be fully operational again.

**Any other remarks**

...
![homey settings](https://cloud.githubusercontent.com/assets/1075840/17690092/9dc45278-638c-11e6-8999-12f5d5064e39.png)

**Comment by Inversion-NL** [comment]
I also had this out of the blue, but luckily I was re-adding my devices after the infamous Z-Wave rewrite so just did a reset of my Z-Wave chip and all was fine after that

**Comment by Corvo** [comment]
Resetting the Z-Wave network is quite a hassle. Every device needs to be reset before being re-added to Homey (well, atleast my Fibaro Wall Plugs required this).
And after that all flows are broken and need to be fixed by re-adding the correct device specific cards :disappointed: 

So, if possible, I'd like to avoid a z-wave network reset. Thanks for the info though :+1:

**Comment by Inversion-NL** [comment]
Yes, I had the outmost luck I was already adding my Z-Wave devices.
Just wanted you and Athom know your not alone in this.

**Comment by Corvo** [comment]
Rebooting didn't solve the problem.
@jeroenvollenbrock suggested shutting Homey down _(pull the plug)_ and start Homey up again _(plugging it back into the socket)_. This *fixed* my problem.

So there is seemingly a difference between a software restart and a manual stop & start. For now I'm happy (I didn't have to reset my Z-Wave network). But not being able to perform a restart (in fear of this problem), still needs an official fix.

If @WeeJeWel or @jeroenvollenbrock need any help and/or logs, let me know!

**Comment by WeeJeWel** [comment]
Pretty odd problem.

Next time it occurs, please report the json response from the `/api/wireless/zwave/state/` call. You can find this in the Network tab in the Chrome Console.

**Comment by Corvo** [comment]
I tried a reboot 3 times... but Homey simply refuses to fail ;)
As soon as I encounter it again, I'll report the Z-Wave state (json).

**Comment by MdeRhoter** [comment]
> Next time it occurs, please report the json response from the /api/wireless/zwave/state/ call.

How do I get to that URL? If I go to http://ipadress/api/wireless/zwave/state/, I get this:

`{"status":401,"result":"no_auth_method_found"}`

**Comment by Inversion-NL** [comment]
Tried it as well and got the same message.
Tried both local as remote (with token etc)

**Comment by WeeJeWel** [comment]
Open the Chrome console:

```
api('GET', '/wireless/zwave/state/', null, function( err, result ) { console.log( err, result ); });
```

**Comment by MdeRhoter** [comment]
That sure gets you a load of stuff!

`Object {ready: true, zw_state: Object, regions: Object}
ready
:
true
regions
:
Object
zw_state
:
Object
CONTROLLER_IS_REAL_PRIMARY
:
false
CONTROLLER_IS_SECONDARY
:
true
CONTROLLER_IS_SUC
:
false
CONTROLLER_NODEID_SERVER_PRESENT
:
true
CONTROLLER_ON_OTHER_NETWORK
:
false
NO_NODES_INCUDED
:
false
activeFirmware
:
"EU"
capabilities
:
Object
api
:
"controller"
controllerType
:
"primary"
timerSupported
:
false
__proto__
:
Object
__defineGetter__
:
__defineGetter__()
__defineSetter__
:
__defineSetter__()
__lookupGetter__
:
__lookupGetter__()
__lookupSetter__
:
__lookupSetter__()
constructor
:
Object()
hasOwnProperty
:
hasOwnProperty()
isPrototypeOf
:
isPrototypeOf()
propertyIsEnumerable
:
propertyIsEnumerable()
toLocaleString
:
toLocaleString()
toString
:
toString()
valueOf
:
valueOf()
get __proto__
:
__proto__()
set __proto__
:
__proto__()
chip_type
:
5
chip_version
:
0
currentCommand
:
"sendData"
currentProcess
:
"ProcessSendData"
homeId
:
84417837
networkKey
:
"0aad0f28e78cd48be042f7ce7190cf00"
nodeId
:
0
nodes
:
Array[4]
powerLevel
:
197
sucId
:
197
ver
:
5
__proto__
:
Object
__proto__
:
Object`

Is there a prettier/easier way of getting that response here?

**Comment by WeeJeWel** [comment]
That's ok. Can you also post a screenshot of the settings view with it?

**Comment by MdeRhoter** [comment]
I sure can!

![image](https://cloud.githubusercontent.com/assets/15713184/17975641/cb81d22a-6aea-11e6-8688-f0a0bb6f9556.png)

**Comment by WeeJeWel** [comment]
Sure looks good to me?

**Comment by MdeRhoter** [comment]
It would look even better if my three Fibaro motion sensors reported any motion they sensed! And should those buttons be greyed out all the time?

**Comment by WeeJeWel** [comment]
Well, you kinda hijacked this issue for another issue. Give 0.9.3 a try when it's out, should solve those grey buttons.

**Comment by MdeRhoter** [comment]
To be fair, I only asked how to perform the /api/wireless/zwave/state/ call - you somehow wanted screenshots.

If 0.9.3 also solves my silent detectors I'll be very happy :)

**Comment by WeeJeWel** [comment]
@Corvo Can you please report back? :)

**Comment by Corvo** [comment]
I haven't needed to restart my Homey since. _(well, there's the auto-restart after the v0.9.3, I guess)_
Currently at an uptime of _895056_.

So @WeeJeWel this issue hasn't resurfaced. The only problem that lingers ever since, is that my Motion Sensor's battery level isn't reported / shown.

**Comment by Corvo** [comment]
Yesterday I had to power-off my Homey (and thus later, started-up again).
Today I noticed that only Lux is being reported...
...so yes: the bug is back.

@jeroenvollenbrock @WeeJeWel It took 16 days to get there... but this is the response returned by `/api/wireless/zwave/state/`


```json
{
  "status": 200,
  "result": {
    "ready": true,
    "zw_state": {
      "region": "AE",
      "regions": {
        "CN": {
          "hardware": "AE",
          "title": "China"
        },
        "EU": {
          "hardware": "AE",
          "title": "European Union",
          "active": true
        },
        "IN": {
          "hardware": "AE",
          "title": "India"
        },
        "MY": {
          "hardware": "AE",
          "title": "Malaysia"
        },
        "RU": {
          "hardware": "AE",
          "title": "Russia"
        }
      },
      "activeFirmware": "EU",
      "ver": 5,
      "capabilities": {
        "api": "controller",
        "timerSupported": false,
        "controllerType": "primary"
      },
      "nodes": [
        1,
        2,
        3,
        4,
        5,
        6,
        7,
        8
      ],
      "chip_type": 5,
      "chip_version": 0,
      "homeId": 84418047,
      "nodeId": 0,
      "CONTROLLER_IS_SECONDARY": true,
      "CONTROLLER_ON_OTHER_NETWORK": true,
      "CONTROLLER_NODEID_SERVER_PRESENT": true,
      "CONTROLLER_IS_REAL_PRIMARY": false,
      "CONTROLLER_IS_SUC": false,
      "NO_NODES_INCUDED": false,
      "sucId": 28,
      "powerLevel": 1,
      "networkKey": "64f15ee194eddccc2b382db4a361780a",
      "methodsAsync": [
        "addNodeAbort",
        "removeNodeAbort",
        "replaceFailedNodeAbort",
        "sendData",
        "isFailedNode",
        "getFailedNodes"
      ]
    }
  }
}
```

**Comment by WeeJeWel** [comment]
There seems to be a temporary fix: Change your region to something else, and change it back.

**Comment by Corvo** [comment]
I already pulled-the-plug to fix the problem. Next time it happens, I will keep that in mind.

I did try this temporary fix: hoping it might fix my problem with the battery-levels (not being reported).
Alas, that issue remains.

**Comment by WeeJeWel** [comment]
Should be fixed in 0.10.4, if not, please report back!
